<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ob</class>
 <widget class="QDialog" name="Ob">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="iesire">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>550</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Iesire</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>781</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="showobstr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>430</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Afiseaza obiectivele strategice la nivel de companie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showobproces">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>430</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Afiseaza obiectivele operationle la nivel de proces</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showobagentie">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>430</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Afiseaza obiectivele operationale la nivel de agentie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addobstr">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>341</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adauga obiectiv strategic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addobproces">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>380</y>
     <width>341</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adauga obiectiv operational pentru procese</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addobagentie">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>420</y>
     <width>341</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adauga obiectiv operational pentru agentii</string>
   </property>
  </widget>
  <widget class="QLabel" name="newob">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pentru adaugarea unui nou obiectiv, completati campurile urmatoare:</string>
   </property>
  </widget>
  <widget class="QLabel" name="def">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Definire obiectiv:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="defEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>771</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="tipBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>771</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="tip">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-- text care se schimba -- vine tip obectivului asociat --</string>
   </property>
  </widget>
  <widget class="QLabel" name="indicator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Indicator:</string>
   </property>
  </widget>
  <widget class="QLabel" name="tinta">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tinta:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="indicatorEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="tintaEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="aferent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-- agentie/proces aferenta --</string>
   </property>
  </widget>
  <widget class="QComboBox" name="aferentBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="salveaza">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>480</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salveaza</string>
   </property>
  </widget>
  <widget class="QPushButton" name="anuleaza">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>520</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Anuleaza</string>
   </property>
  </widget>
  <widget class="QComboBox" name="aferentBox2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
